jsonnet/control-plane.libsonnet

```diff
@@ -24,6 +24,22 @@ function(params)
       },
     },
 
+    prometheusRule+: {
```

Comment:
It would be better to put this with other openshift-specific rules in https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/rules.libsonnet

Reply:
the reason i put it here is because the there are similar recording rules for replicaset etc that are present in this file.

Reply:
Yes, but `replicaset` is a k8s concept, meanwhile `deploymentconfig` is not, hence it might be good to put it directly in `openshift` group.